webtool/app.py

```
@@ -860,6 +861,91 @@ def export_srt(project: str, base: str, sprecher: bool = True):
     return {"srt": srt}
 
 
+def _get_or_render_md(project: str, base: str) -> str | None:
+    """Holt fertiges Markdown oder rendert es aus dem Dokument."""
+    md_p = _md_path(project, base)
+    if os.path.exists(md_p):
+        try:
+            with open(md_p, "r", encoding="utf-8") as fh:
```

Ruff (0.16.2)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)